epsagon/runners/tornado.py

```diff
@@ -29,7 +30,8 @@ def __init__(self, start_time, request):
 
         self.event_id = str(uuid.uuid4())
 
-        self.resource['name'] = 'Tornado'
+        # Since Tornado doesn't has app name, we use the tracer app name.
+        self.resource['name'] = tracer.app_name
```

Comment:
this sounds like a bad idea. is this what we are doing in flask? i think in flask we use the route, thats better (similar to api gateway)

Reply:
Done.